base/verify_node

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# fail fast
+set -e
+
+# Bail with a help message if the arguments look wrong
+if [ $# != 2 -o "${1}" == "help" ]; then
+  echo "Usage: ${0} <node-js-binary-file> <SHASUMS256.txt.asc file>"
+  echo ""
+  echo "Verifies the integrity of the given Node.js binary file using the "
+  echo "given SHASUMS256.txt.asc file."
+  echo ""
+  echo "Note: To be valid, the filename of the given Node.js binary must "
+  echo "      match the expected name in the SHASUMS256.txt.asc file."
+  exit 1
+fi
+
+# Use absolute paths for safety
+GREP='/bin/grep'
+CAT='/bin/cat'
+GPG='/usr/bin/gpg'
+RM='/bin/rm'
+SHASUM='/usr/bin/shasum'
```

Comment:
nit: drop these variables. If the paths ever change, this file will break. Let's rely on the distribution / base image having these in the default $PATH.